RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 -----
 * [*] [internal] Refactored fetching posts in the Reader tab, including post related operations (i.e. like/unlike, save for latter, etc.) [#20197]
 
```

Comment:
I don't think we need this empty line, right?

```suggestion
```

Reply:
No we don't. git added this line when merging the trunk branch and I didn't notice it 🙈